src/arch/host/include/arch/atomic.h

```diff
@@ -13,6 +13,8 @@
 
 #include <stdint.h>
```

Comment:
Please use a prefix for your commit message.

E.g include: `Fix zephyr conflicting types`

then please explain in the commit message WHY do we need this change. Do you get a compiler error? 

Please give more information so that people can understand the intention of your change.